src/planscape/config/conditions.json

```diff
@@ -1227,7 +1227,7 @@
                 {
                   "display_name": "Potential Climate Refugia (Late Century)",
                   "data_provider": ": Information Center for the Environment, UC Davis",
-                  "data_units": "Percent",
+                  "data_units": "Continuous",
```

Comment:
just to be sure, is it "Continuos" ? I see others as  `"data_units": "Continuous Variable",`